contentgrid-appserver-query-engine-impl-jooq/src/main/java/com/contentgrid/appserver/query/engine/jooq/JOOQThunkExpressionVisitor.java

```diff
@@ -195,6 +199,17 @@ public Field<?> visit(FunctionExpression<?> functionExpression, JOOQContext cont
         return result;
     }
 
+    @Allow.PlainSQL
+    private Condition generateFTSCondition(@NonNull JOOQContext context, @NonNull ContentGridFullTextSearch contentGridFullTextSearch) {
+        var left = contentGridFullTextSearch.getLeftTerm().accept(this, context);
+        var right = contentGridFullTextSearch.getRightTerm().accept(this, context);
+
+        var locale = contentGridFullTextSearch.getLocale();
+        var language = locale.getDisplayLanguage(ENGLISH);
+
+        return DSL.condition("to_tsvector(?, coalesce(?, '')) @@ websearch_to_tsquery(?)", DSL.inline(language), left, DSL.inline(right));
```

Comment:
JOOQ handles `Field<?>` correctly, no need to do `DSL.inline()`.
```suggestion
        return DSL.condition("to_tsvector(?, coalesce(?, '')) @@ websearch_to_tsquery(?)", DSL.inline(language), left, right);
```